acceptance/bundle/includes/glob_in_root_path/[abc]/databricks.yml

```diff
@@ -0,0 +1,2 @@
+include:
+  - "*.yml"
```

Comment:
is the idea here to test that resource.yml is included correctly? might be worth a comment then.

Reply:
It was just there to have a non-empty databricks.yml. I've removed it since it is unnecessary for the test assertion.